pkg/operator/cluster.go

```diff
@@ -157,18 +223,9 @@ func (t *clusterTPR) Watch() error {
 
 			switch event.Type {
 			case kwatch.Added:
-				ns := c.Spec.Namespace
-				if ns == "" {
-					logger.Errorf("missing namespace attribute in rook spec")
-					continue
-				}
-
-				newCluster := cluster.New(c.Spec, t.context.factory, t.context.clientset)
-				t.tracker.add(ns, c.Metadata.ResourceVersion)
-				t.clusters[ns] = newCluster
 
-				logger.Infof("starting new cluster %s in namespace %s", c.Metadata.Name, ns)
-				t.startCluster(newCluster)
+				logger.Infof("starting new cluster in namespace %s", c.Name)
```

Comment:
is `c.Name` actually the namespace here (like the comment says)? or is it the name of the cluster?  maybe log both if so